R/support.R

```diff
@@ -362,17 +362,34 @@ combine_rt <- function(rt, samples) {
 }
 
 
+combine_rt_EpiEstim <- function(rt, samples) {
```

Comment:
This function is never used so can be removed

Reply:
have added a test for this